.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discord
+    url: https://discord.gg/dgRawPdNuC
+    about: If you have any questions, feel free to join our Discord server.
```

Comment:
Currently, there's no template for questions but a discord link instead.
Should I add on?